### DISCUSSION TITLE
イベントブースの備品作成の相談

### DISCUSSION THREAD

**Discussion by gurezo** [mozilla-japan/translation#756] [General]
### 作成したい備品
- テーブルクロス
- ラミネートされた QR コードとアイコン
  - X(旧Twitter)
  - 翻訳ガイドブック
  - Translation リポジトリ

### 購入可能な備品

- テーブルクロス留め
  - https://www.az-nobori.com/goods/411725
- イーゼル
  - mozilla 台湾のグッズにあるかも？

### TODO

- 制作費用算出
  - サイズとデザインを決める @gurezo 
- テーブルクロスのデザイン @gurezo 
  - https://images.app.goo.gl/BF5KAEQv8icPH7Nh8 （個人的デザインイメージ）
- OSC の机のサイズ
  - W1,500×D600×H700mm
- mozilla 台湾の方に制作費用や前例の有無を相談する @hmatrjp 

### 課題

- 保管場所
- テーブルクロス制作の前例
- 制作費用
  - カンパ？
  - 他の選択肢の有無
  - https://www.noboriprint-u.com/page/tablecloth_design

- 納期

**Comment by gurezo** [comment]
memo
CHIRIMEN Hardware コミュニティのテーブルクロスは、5 年以上使って、初めて洗濯の予定

**Comment by elie-j** [comment]
@gurezo @potappo  MDNコミュニティとして、イベント出展用のテーブルクロス制作されるのであれば制作費の部分は、WebDINOで支援出来ると思います。

**Reply by gurezo** [reply]
cf. https://github.com/mozilla-japan/translation/discussions/756?sort=new#discussioncomment-9638952

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 

入稿データの作成をチャレンジしましたが、断念しました。
別途相談させてください。 :bow:

cf. https://github.com/mozilla-japan/translation/discussions/756#discussioncomment-9717297

**Comment by gurezo** [comment]
@hmatrjp @potappo @igrep. cc @elie-j 

デザイン案と参考価格を出してみました。
価格は正確な見積もりを出したわけではないので、参考価格になります。
デザインは、ざっくりと Excel にて作成しました。

`赤`を入れた理由は日の丸の`赤`を使いたく入れてみました。

## デザイン案

### 素材

[MDN_Web_Docs.svg/1024px-MDN_Web_Docs.svg.png](https://ja.wikipedia.org/wiki/%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB:MDN_Web_Docs.svg)

### l10n 国旗入りバージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-13 22 43 20](https://github.com/mozilla-japan/translation/assets/7631567/4487146c-dad1-4ef9-b484-ade2f2b08ac6)

### Japan Community バージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-13 22 43 45](https://github.com/mozilla-japan/translation/assets/7631567/5d598e8a-98bc-4d4e-bb10-8ea90e9dd937)

### Japan localization バージョン x 2(白黒`赤`、白黒)
![スクリーンショット 2024-05-13 22 44 07](https://github.com/mozilla-japan/translation/assets/7631567/32505c98-102a-4385-a16e-f9c6749a42c2)


## 参考価格

- [テーブルクロス前掛けタイプのデータ制作プラン ](https://www.noboriprint-u.com/page/tablecloth_apron_design)

![スクリーンショット 2024-05-13 22 54 58](https://github.com/mozilla-japan/translation/assets/7631567/fe8647f9-b6d7-4c28-8948-a0803d8558b1)

**Comment by elie-j** [comment]
@gurezo  こちらちょっとだけ気になったので、コメントを・・・
今回は「前垂れタイプ（横がなくて前だけを隠すやつ）」のテーブルクロスを作る予定でしょうか？　CHIRIMENコミュニティのテーブルクロスや弊社の他のプロジェクトで作ったイベント用クロスなどもそうですが、使用する机の奥行きなどは会場によって異なるので（奥行 45cmの机だったり、60cmくらいある奥行きの机だったり）、汎用性を考えると普通のフラットタイプのものを作っておいた方が良いのではないかなー？と思います。（たぶん普通のクロスより前垂れタイプの方が安いのでこちらで考えられたのだと思いますが、費用面は今回弊社で支援しますので、せっかくなので汎用性の高いものを作っておいた方が良いと思います。）

デザインの入れる位置などは別の取り組みで作ったラフイメージがあるのでそれを参考までにDMでお送りしておきますね。

あと、1点だけお願いですが、弊社で費用支援する場合は、弊社で直接業者さんに発注して弊社でカード決済する必要があるので、デザインデータいただいて弊社から発注になるのでそこだけご注意いただけばと思います。

**Reply by gurezo** [reply]
メモ
サイズは大きい方、某イベントのテーブルクロスのサイズにすることになりました。

**Comment by gurezo** [comment]
下記 discussion で、デザイン投票中
https://github.com/mozilla-japan/translation/discussions/762

締切は 2024.05.25（土）15:00 まで

**Reply by gurezo** [reply]
[#766](https://github.com/mozilla-japan/translation/discussions/766) 2024.05.28(火) 23:59 締め切り

**Reply by gurezo** [reply]
1. Japanese Localization (日本語ローカライズ) 白文字 で、決定

![MDN-Web-Docs-Japanese Localization white](https://github.com/mozilla-japan/translation/assets/7631567/7787f46e-a84e-4a95-b3b5-58634d972502)

**Comment by gurezo** [comment]
### デザイン資料
[MDN.Web.Docs.Jp.part1.xlsx](https://github.com/mozilla-japan/translation/files/15360732/MDN.Web.Docs.Jp.part1.xlsx)

**Comment by gurezo** [comment]
@hmatrjp @potappo cc @elie-j 

決まったデザインを基にデーブルクロスのサイズの見取り図を書いてみました。
お時間のある時に確認とアドバイスをお願いします 🙇 

[MDN.Web.Docs.テーブルクロス.pdf](https://github.com/user-attachments/files/15525766/MDN.Web.Docs.pdf)
[MDN.Web.Docs.テーブルクロス.xlsx](https://github.com/user-attachments/files/15525767/MDN.Web.Docs.xlsx)

**Reply by gurezo** [reply]
@hmatrjp @potappo 
今回デザインで作成したイラレファイルやエクスポートした画像は、どこかのリポジトリにプッシュした方が良いでしょうか？
現在は自分のローカルに保存してあります。

**Reply by hmatrjp** [reply]
デザインの作成ありがとうございます。 @gurezo 
PDFと同じように、こちらに添付してもらえれば、それで用が足りるような気もしますが、どうでしょう？ @potappo 

私からは、特に「ここを直さないと」という点はありません。机の寸法の話がクリアできていれば、それで良いと思います。

もし、校正など、変更（バージョンアップ）していくことを想定するなら、ひとつリポジトリを作っても良いと思います。
その場合は、[MDN](https://github.com/mozilla-japan/translation/tree/master/MDN) の下で、textlint と同レベルに一つディレクトリを作ってその中でしょうか。

**Reply by gurezo** [reply]
@hmatrjp 
イラレファイルは、容量の問題で、スレッドに添付できなかったです。。。。。
ロゴの画像自体は、添付できます。

素人加工なので、本職のデザイナーさんから見たらツッコミ多数な出来栄えなので、ちょっと恥ずかしいかもです。

**Reply by hmatrjp** [reply]
容量の問題とは気づきませんでした。 私はイラストレーターにほぼ縁がない人間で、
> ツッコミ多数な

がピンとこないレベルですので、 なんともですが、@gurezo さんの出せる範囲で出してもらえればいいと思います。

出稿した後に、業者から微修正の要求がある可能性は、ちらっと頭を掠めた(*)ので、どちらかというとその関係かと思っていました。
*) 別の場所で、印刷業者とのやりとりで前例あり

**Reply by gurezo** [reply]
本職のデザイナーさんだと、 1px のズレも許さない人達だと思ってるので（個人的イメージ＆冗談）気にしてみました。

入稿データに不備があると訂正求められますよね。。。。
イラレデータは一旦自分のリポジトリに全て集めます。

**Reply by gurezo** [reply]
@hmatrjp @potappo 
[MDN.Web.Docs.テーブルクロス.pdf](https://github.com/user-attachments/files/15525766/MDN.Web.Docs.pdf) に問題がなければ、入稿のプロセスに進めても良いですか？

**Reply by potappo** [reply]
@gurezo 
入稿データのフォーマットには詳しくないので、ぱっと見の見た目でしか判断できないですが、イメージとしては問題ないと思います。

**Comment by hmatrjp** [comment]
はい、よろしくお願いします。 @gurezo
一応、縦横寸法の検算だけはしました。

Message ID: <mozilla-japan/translation/repo-discussions/756/comments/9651849
@github.com>
>
>

**Comment by elie-j** [comment]
@gurezo デザイン案が確定したようですが、[以前こちらのスレッド](https://github.com/mozilla-japan/translation/discussions/756#discussioncomment-9469352)でお願いしたように、弊社で制作費支援を行う場合は、弊社から業者さんに直接発注する必要があるので、入稿用のデータができたら発注内容（サイズや素材など（防炎タイプ or not 防炎タイプとかあるはずなので）の情報私と篠田宛にお送りいただけると助かります。
もしも入稿用にトンボ付データ作るのが大変などあれば、Slackの方に直接ご相談いただければと思います！

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 
決定稿をこちらにアップします。
※イラレデータは駄目でしたが、 zip にまとめたら大丈夫でした。
![MDN-Web-Docs-Japanese Localization white](https://github.com/mozilla-japan/translation/assets/7631567/1e4fedcd-e87f-4fa7-b932-a4beb834500c)

### Design Zip
[2024.05.29.final.design.zip](https://github.com/user-attachments/files/15750489/2024.05.29.final.design.zip)

**Comment by gurezo** [comment]
@hmatrjp @potappo 

デザインデータは、[こちら](https://github.com/mozilla-japan/translation/discussions/756#discussioncomment-9717297)にアップしました。
ボツデザインに関しては、自分のプライベートリポジトリにプッシュしました。
必要であれば、お二人を invite しようと思います。

**Reply by gurezo** [reply]
@hmatrjp @potappo 
お二人をプライベートリポジトリに invite しました。
お時間ある時に確認をお願いします。
今月(7月)のミートアップの時に諸々相談させてください

**Reply by gurezo** [reply]
一旦保留

**Comment by gurezo** [comment]
@hmatrjp @potappo cc @elie-j 

ロゴが新しくなっていると教えて頂きました。
今回のデザインは、一旦保留にしたいと思います！
第70回のミートアップで、相談させてください！

## 2020年に新しくなっている

- https://www.zdnet.com/article/mozilla-unveils-redesign-of-mdn-platform-new-logo/
- https://qiita.com/Naughty1029/items/27caf6276167187e127a
- https://ja.wikipedia.org/wiki/%E3%83%95%E3%82%A1%E3%82%A4%E3%83%AB:MDN_Web_Docs_logo.svg
- リポジトリはこれ？
  - https://github.com/mdn/yari/tree/main/client/src/assets
    - [mdn-logo.svg](https://github.com/mdn/yari/blob/main/client/src/assets/mdn-logo.svg)

**Reply by potappo** [reply]
@gurezo 
デザイン保留、了解です。

MDN サイトで実際表示されているロゴはグラデーションしていますね。

https://developer.mozilla.org/

（技術的に少し気になったので）
SVG のグラデーションについては、以下にドキュメントがあります。
https://developer.mozilla.org/ja/docs/Web/SVG/Tutorial/Gradients

元々のデザインというわけではなく、去年の pride month のために変わって 約 1 年そのままのようです。参考までに。
https://github.com/mdn/yari/pull/8979

**Reply by hmatrjp** [reply]
それでは、6/22のミートアップの時に。
@gurezo すみません、もらったプライベートリポジトリへの招待が期限切れになってしまいました。

**Reply by gurezo** [reply]
@hmatrjp 今 invite しました！

**Reply by gurezo** [reply]
比較のために参考アップロード

![mdn-docs-logo](https://github.com/mozilla-japan/translation/assets/7631567/d94eb44a-73f8-4e1b-a817-ef5ee8020e4a)
![MDN-Web-Docs-Japanese Localization white](https://github.com/mozilla-japan/translation/assets/7631567/fbccda35-7088-42b1-bfe0-9fc7f8674385)

**Reply by gurezo** [reply]
2024.06.22 中間報告で、新デザインでロゴを作ることが決定

**Reply by gurezo** [reply]
最終デザイン案
![M-MDN-web-docs](https://github.com/mozilla-japan/translation/assets/7631567/77f35e8c-73e9-4faf-ba92-759a7fc32e33)

**Reply by gurezo** [reply]
@hmatrjp @potappo @igrep cc @elie-j 

問題なければ、 👍 のスタンプをお願いします！

### テーブルクロス資料

- [MDN.Web.Docs.テーブルクロス2024.06.22.pdf](https://github.com/user-attachments/files/15935912/MDN.Web.Docs.2024.06.22.pdf)
- [MDN.Web.Docs.テーブルクロス2024.06.22.xlsx](https://github.com/user-attachments/files/15935914/MDN.Web.Docs.2024.06.22.xlsx)

### デザイン
![M-MDN-web-docs](https://github.com/mozilla-japan/translation/assets/7631567/dd8ee5f5-45eb-4464-a4ed-5633c818d2a5)

**Reply by gurezo** [reply]
@hmatrjp @potappo @igrep cc @elie-j 

【共有】
2024.06.22 18:41 に @elie-j  さんへテーブルクロスの発注以来済みです。

**Reply by elie-j** [reply]
@gurezo  cc @hmatrjp @potappo 
テーブルクロス、WebDINO で入稿用データを作成して業者さんに発注しますが、すみません、一点だけ気になってしまったので、コメントさせてください。
[MDNの新しいロゴデータのベクターデータ](https://upload.wikimedia.org/wikipedia/commons/1/16/MDN_Web_Docs_full_logo.svg) を確認したところ、本家で提供されているロゴデータは、頭(M)とお尻の部分( _ ) にブルーを使ったデザインなので、このまま色を改変せずに（黒背景なので、フォントの黒文字の部分を白文字にするだけ）で使わせてもらえないでしょうか。

おそらくこのブルーを使うという点（場合によってはミントグリーンを使ったりすることもあるようですが）がデザインの特徴的なところだと思うので、特段大きな理由がない限りは、公開されているロゴのデータの色を変えたりしないでそのまま使った方が問題がないように思います。

今回は黒字の生地にMDNの新しいロゴを配置して、下の段に　Japanese Localization と白い文字で記載するデザインというご指定なので、一旦、MDNの本家ロゴ部分は極力そのままにしてJapanese Localization という文字を配置したデータを作ってみるのでご検討いただければと思います。
（その方がMDN webdocs のサイトと同じようなテイストになるかと・・・）
企業やデザイナーさんによってはロゴの色をいじられるのを嫌がるケースもあって、ちょっと気になってしまったので(^^;)

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 

コメントありがとうございます。

> 本家で提供されているロゴデータは、頭(M)とお尻の部分( _ ) にブルーを使ったデザインなので、このまま色を改変せずに（黒背景なので、フォントの黒文字の部分を白文字にするだけ）で使わせてもらえないでしょうか。

自分個人は特に異論はありません。
@hmatrjp @potappo 如何でしょうか？

このコメントで、本家と同じように「Ｍ」と「＿」の文字の色を青？水色？に変更して欲しいという理解であってますか？
※要カラーコード~~確認~~ : #1870f0;

<details>
<summary>カラーコード</summary>

![スクリーンショット 2024-07-22 220540](https://github.com/user-attachments/assets/bd754f6d-6dd6-475c-9fdf-98c6145a84df)

</details>

この対応の期日・希望日などあったら教えて下さい。

**Reply by potappo** [reply]
@gurezo 
私も特に異論はないです。

**Reply by elie-j** [reply]
@gurezo さん、@potappoさん、お返事ありがとうございます。
ちなみに、この色をブルーにする対応（デザインデータ作成）は、WebDINOで行うので、 @gurezo さんに何か作業していただく必要はありません。
ブルーの入った入稿データができたら、こちらで共有させていただきますね。少しお時間いただければ。

**Reply by hmatrjp** [reply]
出遅れましたが、私も異論ありません。 @gurezo , @potappo , @elie-j 
デザインから入稿の手続きまで、ほぼお任せになってしまっていますが、よろしくお願いします。

**Reply by elie-j** [reply]
@gurezo @potappo @hmatrjp ご連絡が遅くなりましたが、弊社の篠田さんに入稿用データを作ってもらいました。
入稿データはイラストレータファイルで作っているので確認用にPDF変換したデータをこちらに貼りますね。

MDN本家のロゴデータはそのままにして、下の段に "Japanese Localization" というテキストを配置していますが、この場合、上の段の "mdn web docs" という文字よりも下の段の文字が大きくなってしまうとデザイン的にバランスが悪くなるように思いました。
なので、本家ロゴの "mdn web docs" よりも下の段の文字が大きくならないよう【B案】も作成してもらいました。

[A案-テーブルクロス入稿データ確認用.pdf](https://github.com/user-attachments/files/16439437/A.-.pdf)

[B案-テーブルクロス入稿データ確認用.pdf](https://github.com/user-attachments/files/16439440/B.-.pdf)


「いやいや "Japanese Localization" を大きく主張すべきだ！」ということであれば【A案】で作成しますが、 @gurezo さん、 @potappo さん、 @hmatrjp さんのご意見お聞かせいただければと思います。

**Reply by hmatrjp** [reply]
入稿データの作成ありがとうございます。 @elie-j 
私としては、A, Bどちらでもよい　に1票です。　A案も、上段の幅から下段がはみ出してはいないので、不自然に感じていませんでした。
（どうしても A か B かどちらか選べと言われれば、Aに。）

両案とも、1,800 が 1.800　と（カンマでなく）ピリオドを使って表記されていますが、これはこのままでも大丈夫ですか？

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 

基本的に A 案で、自分も良いと思います。

> 両案とも、1,800 が 1.800　と（カンマでなく）ピリオドを使って表記されていますが、これはこのままでも大丈夫ですか？

この表記ミス？以外は、特に無いです。

**Reply by potappo** [reply]
@elie-j
私も A 案で問題ないです。B 案のようにバランス取るのもありかと思いますが、元々の A 案で良いと思います。

**Reply by elie-j** [reply]
@hmatrjp @gurezo @potappo 早速ご確認いただきありがとうございます！ではA案で準備をすすめます。1.800のカンマではなくピリオドの誤記については私も見落としていました！教えていただきありがとうございます。こちらの修正を行った上で、最終的なイラストレーターの入稿データをGithubにアップさせていただきますね！みなさん、お忙しいところありがとうございましたー m(_ _)m

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 
よろしくお願い申し上げます。m(__)m

**Comment by elie-j** [comment]
@gurezo @hmatrjp @potappo またまたその後のご連絡が遅くなりましたが、業者さんに発注しました。入稿データですが、こちらで作成したデータがRGBモードのままだったようで（スミマセン弊社内の確認不足です）、印刷はCMYKモードに変換する必要があると業者さんから指摘を受けたため、最終、以下が入稿データ（確認用jpeg) になります。

最終版の入稿データですが、Github 見てみたのですが、色々ファイルが入っていてどこに最終版をアップすればよいのかちょっとわからなかったので、aiファイルの最終版は @gurezo さんに別途共有させれいただきますね！

![mdm_テーブルクロス(確認用データ)_cmyk](https://github.com/user-attachments/assets/efc9a998-f6c0-4c86-8361-ed9831a8b4de)

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 
共有有難うございます！
後ほど、mozilla-japan の organization に今回のデザインデータのリポジトリを作成します！

**Reply by gurezo** [reply]
@hmatrjp @potappo cc @elie-j 
今ロゴデザインのリポジトリを作成しました！
 リポジトリ名や README に過不足があれば、 Pull Request をお願いします！
[community-logo](https://github.com/mozilla-japan/community-logo)

**Reply by potappo** [reply]
@gurezo 
作成ありがとうございます。community-logo だと、外から見た人がイメージするのは富士山だと思うので、mdn-community-logo の方が良いのではと思います。mdn で始まるリポジトリは既にいくつかあるので、グループ化する意味でも mdn が頭にあって良いと思います。

**Reply by gurezo** [reply]
@potappo 
有難うございます！
[mdn-community-logo](https://github.com/mozilla-japan/mdn-community-logo) に変更しました！

**Reply by gurezo** [reply]
@elie-j cc @potappo @hmatrjp 
テーブルクロスの納期を教えて下さい！
今日のミートアップで、OSC Tokyo 2024 Fall の参加の話題が出たので確認させて下さい！ 🙇 
必要機材は別途調整の連絡をさせて下さい。

**Reply by elie-j** [reply]
@gurezo テーブルクロスは8/29には納品される予定です。
イベント備品の準備や発送は、出張やテレワークですぐに対応出来ない時もあるので、決まったら必要なものなど早めにお知らせいただければ準備するようにします。m(_ _)m

**Reply by gurezo** [reply]
@elie-j cc @hmatrjp @potappo 
次回のミートアップが、2024.09.21（土）なので
その時に詳細な発送備品を連絡させて頂きたいと思います。

以来は、こちらの「[OSC2024 東京秋（10/26 リアル展示）へのブース出展について #786](https://github.com/mozilla-japan/translation/discussions/786)」で、連絡させて頂きたいと思います。 🙇

**Comment by gurezo** [comment]
テーブルクロスが完成したので、close
